### PR TITLE
Make level 1 in autopark project.

### PR DESCRIPTION
**Pull request by aces47**

### CODE REVIEW

File: AutoParkConsole/Classes/VehicleType.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AutoParkConsole.Classes
+{
+	class VehicleType
+	{
+		string _TypeName;
+		double _TaxCoefficient;
```
Comment (PushMeNow) [inline AutoParkConsole/Classes/VehicleType.cs:12]: You have to always write access modifiers in any cases.
Also names of privavate fields have to look like camel case: _privateField

File: AutoParkConsole/Program.cs [hunk 2]
```diff
@@ -1,12 +1,45 @@
...
 		{
-			Console.WriteLine("Hello World!");
+			VehicleType[] vehicleTypes =
+				{ new("Bus", 1.2d), new("Car", 1d), new("Rink", 1.5d), new("Tractor", 1.2d) };
+
+			double averageTax = 0d;
+			double max = vehicleTypes[0].GetTaxCoefficient();
+			double currentTax;
+
+			for(int counter=0; counter<vehicleTypes.Length;counter++)
```
Comment (PushMeNow) [inline AutoParkConsole/Program.cs:17]: Spaces between statements

**Review by PushMeNow — CHANGES_REQUESTED** [review]

**Review by UNDogMan — CHANGES_REQUESTED** [review]
Clean your code with code conventions.

File: AutoParkConsole/Program.cs (hunk 2 above)
Comment (aces47) [reply AutoParkConsole/Program.cs]: Fixed. Thank you!

File: AutoParkConsole/Classes/VehicleType.cs (hunk 1 above)
Comment (aces47) [reply AutoParkConsole/Classes/VehicleType.cs]: Fixed. Thank you!

**Review by PushMeNow — APPROVED** [review]

**Pull request merged**